lib/tasks/say.js

```diff
@@ -201,7 +203,7 @@ class TaskSay extends TtsTask {
   }
 
   async exec(cs, {ep}) {
-    const {srf, accountSid:account_sid} = cs;
+    const {srf, accountSid:account_sid, target_sid:callSid} = cs;
```

Comment:
this actually looks backwards.  You are trying to extract `target_sid` from `cs`, which does not exist, and assign it to callSid.  This tells me also that you have not tested this properly.  Please test and verify the data is being written to the time-series database

Reply:
My bad, apologies for this, as I was not able to generate this scenario in the system it got missed, I have corrected it now, also added missing entries for STT_FAILURE alerts